src/vt-tv/render/render.cc

```diff
@@ -63,7 +63,7 @@ Render::Render(Info in_info)
   grid_size_[2] = 1; // we assume 2D representation
 
   int sqrt_n_ranks = std::sqrt(n_ranks_);
-  if(sqrt_n_ranks * sqrt_n_ranks == n_ranks_) {
+  if(static_cast<uint64_t>(sqrt_n_ranks) == n_ranks_) {
```

Comment:
I think both the older way and the way you suggest doing are error prone; 
Here is what I believe to be the most robust approach:
```c++
uint64_t sqrt_n_ranks = static_cast<uint64_t>(std::sqrt(static_cast<double>(n_ranks_)));
if (sqrt_n_ranks * sqrt_n_ranks == n_ranks_) { // n_ranks_ is perfect square }
```

Reply:
Oh yes I have introduced some error here while fixing a clang error...
Your solution is working. It has been applied.